src/Components/Form.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+
+const Form = () => {
+  
+  const [formData,setForm] =React.useState({
+    firstName:'',
+    lastName:'',
+    
+})
```

Comment:
You can use destructuring:
```suggestion
  const [{firstName, lastName},setForm] =React.useState({
    firstName:'',
    lastName:'',
    
})
```